### DISCUSSION TITLE
How to import google fonts local?

### DISCUSSION THREAD

**Discussion by jk0ehn** [neuefische/questions#263] [Web] [answered]
Hey. Any opinions? thx for feedback

**Comment by mbosselmann** [comment, marked as answer]
Hi Julian,
you need to do the following steps:
- Download the font
- Save the font in your `public` folder (optional: in an extra `font` folder)
- Import this font by using `@font-face` in your `GlobalStyles`. See this example:

```css
	@font-face {
		font-family: FuzzyBubbles;
		src: url('/fonts/FuzzyBubbles-Regular.ttf') format('truetype');
	}
```
- Then you can reference local font 🎉 

For MUI you need to do the following:
- In your `_app.js` you need to import `createTheme` and `ThemeProvider` by MUI
- Create a theme and add the theme to your app by using the ThemeProvider. See this example:

```js
import {createTheme, ThemeProvider} from '@mui/material/styles';

import {GlobalStyle} from '../styles';

export let theme = createTheme({
	typography: {
		fontFamily: 'FuzzyBubbles, sans-serif',
	},
});

export default function App({Component, pageProps}) {
	return (
		<ThemeProvider theme={theme}>
			<GlobalStyle />
			<Component {...pageProps} />
		</ThemeProvider>
	);
}
```

Feel free to ask me if you have questions. :)

**Reply by jk0ehn** [reply]
It works, thank you!
